doc/classes/RichTextLabel.xml

```diff
@@ -56,7 +56,7 @@
 			<return type="void" />
 			<description>
 				Clears the tag stack.
-				[b]Note:[/b] This method will not modify [member text], but setting [member text] to an empty string also clears the stack.
+				[b]Note:[/b] This method will not modify [member text] (although it will hide it until redrawn), but setting [member text] to an empty string also clears the stack.
```

Comment:
Not future tense because the action happens exactly when the method is called.
```suggestion
				Clears the tag stack, causing this label to display nothing until [member text] is modified again.
				[b]Note:[/b] This method does not affect [member text]. However, setting [member text] to an empty [String] also clears the stack.
```
When worded like this, it does sound a bit... odd?

Reply:
> Clears the tag stack, causing this label to display nothing until [member text] is modified again.

But here's the thing, that not exactly true. For example: calling `clear()` before the node is added to the scene will cause the text to show again after.

Reply:
Did my own spin of it, what do you think?